doc/lightning-delpay.7.md

```diff
@@ -0,0 +1,83 @@
+lightning-delpay -- Command for removing a completed or failed payment
+============================================================
+
+SYNOPSIS
+--------
+
+**delpay** *payment_hash* \[status\]
```

Comment:
Escape the underscore (as well all instances)

```
*payment\_hash*
```

Reply:
Changed https://github.com/ElementsProject/lightning/commit/627a374820dd03876eae9e4a99d661cb39f63934